dev-lang/lua/lua-5.3.3.recipe

```diff
@@ -0,0 +1,93 @@
+SUMMARY="A powerful, fast, light-weight, embeddable scripting language"
+DESCRIPTION="
+Lua combines simple procedural syntax with powerful data description \
+constructs based on associative arrays and extensible semantics. Lua is \
+dynamically typed, runs by interpreting bytecode for a register-based virtual \
+machine, and has automatic memory management with incremental garbage \
+collection, making it ideal for configuration, scripting, and rapid prototyping.
+"
+HOMEPAGE="http://www.lua.org"
+SOURCE_URI="http://www.lua.org/ftp/lua-$portVersion.tar.gz"
+CHECKSUM_SHA256="5113c06884f7de453ce57702abaac1d618307f33f6789fa870e87a59d772aca2"
+LICENSE="MIT"
+COPYRIGHT="1994-2015, Lua.org, PUC-Rio"
+REVISION="1"
+ARCHITECTURES="x86 x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	lua$secondaryArchSuffix = $portVersion compat >= 5.3
+	cmd:lua$secondaryArchSuffix = $portVersion compat >= 5.3
+	cmd:lua5.3$secondaryArchSuffix = $portVersion compat >= 5.3
+	cmd:luac$secondaryArchSuffix = $portVersion compat >= 5.3
+	cmd:luac5.3$secondaryArchSuffix = $portVersion compat >= 5.3
+	lib:liblua$secondaryArchSuffix = $portVersion compat >= 5.3
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	lua${secondaryArchSuffix}_devel = $portVersion compat >= 5.3
+	devel:liblua$secondaryArchSuffix = $portVersion compat >= 5.3
+	"
+REQUIRES_devel="
+	lua$secondaryArchSuffix == $portVersion
+	"
+
+PATCHES="lua-$portVersion.patchset"
+if [ $effectiveTargetArchitecture = x86_gcc2 ]; then
+	PATCHES+="
+		lua-$portVersion.gcc2.patchset
+	"
+fi
+
+
+BUILD()
+{
+	make haiku MYCFLAGS="-DLUA_COMPAT_5_1 -DLUA_ROOT='\"$prefix/\"' -DLUA_LDIR='\"$dataDir/lua/5.3/\"'" \
+		LUA_T="lua5.3" LUAC_T="luac5.3" LUA_SO="liblua.so.5.3"
```

Comment:
Same thing with these 2:
<command-line>:0:28: error: too many decimal points in number
luaconf.h:202:18: note: in expansion of macro 'LUA_ROOT'
 #define LUA_CDIR LUA_ROOT "lib/lua/" LUA_VDIR "/"

So keeping in what is in for now :)